src/core/manager.c

```diff
@@ -905,6 +905,14 @@ int manager_new(UnitFileScope scope, ManagerTestRunFlags test_run_flags, Manager
                         return r;
         }
 
+#if HAVE_LIBBPF
+        if (MANAGER_IS_SYSTEM(m) && lsm_bpf_supported()) {
```

Comment:
this block should probably be in the next if? We probably don't want to run this in systemd-analyze

Reply:
I'm not very familiar with this code. Do you mean above in the `else` block? Calls in `src/analyze` set the `MANAGER_TEST_RUN_MINIMAL` so I think that makes sense but maybe I'm missing something.

Reply:
yeah they both check `test_run_flags`, the above block also looks fine to place it